FixtureGroupInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/*
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
```

Comment:
We no longer add license blocks to the top of files since the license in the root folder is valid without this.